seatunnel-e2e/seatunnel-spark-e2e/seatunnel-connector-spark-tidb-e2e/src/test/resources/tidb/tidb_source_and_sink.conf

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+env {
+    # You can set spark configuration here
+    spark.app.name = "SeaTunnel"
+    spark.executor.instances = 2
+    spark.executor.cores = 1
+    spark.executor.memory = "1g"
+    spark.master = local
+    job.mode = "BATCH"
+    spark.tispark.pd.addresses = "127.0.0.1:2379"
+    spark.sql.extensions = "org.apache.spark.sql.TiExtensions"
+}
+
+source {
+
+    tidb {
+        database = "test"
+        pre_sql = "select * from test.e2e_table_source"
+        result_table_name = "spark_test"
+    }
+}
+
+transform {
+
+}
+
+sink {
+
+    tidb {
+        addr = "127.0.0.1"
```

Comment:
use `spark_e2e_tidb`?

Reply:
Yes, I expect so. But now I come across a problem. It seems that I can't deploy tidb in a single docker and connect to it in the way of tispark. It lacks pd and tikv. Do you have any good suggestions?

Reply:
You can use the container to start all dependencies

reference
https://github.com/apache/incubator-seatunnel/pull/2830/files#diff-813f5496f69f3fe1d81254c15504b8c5ea36ec8f9fb2bd90c600429de401a3daR72